app/controllers/users_controller.rb

```diff
@@ -41,8 +41,20 @@ def index; end
 
   def profile; end
 
+  def api_button
```

Comment:
nit: this is a weird name for a controller action, maybe something like `generate_api_key`? Also, what happens if there is already an api key, does this then regenerate it? might that be a bad experience for the user of the API?